react-backend/routes/qr_generator.js

```diff
@@ -1,17 +1,31 @@
 const router = require('express').Router();
 const userRegister = require('../database/queries/user_register');
+const Joi = require('joi');
+const { validate } = require('../shared/middleware');
 const qrcodemaker = require('../functions/qrcodemaker');
 const hash = require('../functions/hash');
 const sendQrCode = require('../functions/qr_send');
 
-router.post('/', (req, res, next) => {
+const schema = {
+  body: {
+    formSender: Joi.string().required(), 
+    formPhone: Joi.number().required(), 
```

Comment:
What about phone numbers that begin with 0? Or that start with `++`? Or have dashes?